CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue
+ or any other method with the owners of this repository before making a change. 
```

Comment:
Let's remove "or any other method..."
We will discuss everything through issues only.

Reply:
ok on it